### PR TITLE
Update apple

### PR DESCRIPTION
**Pull request by EpLiar**

### CODE REVIEW

File: data/beats [hunk 1]
```diff
@@ -708,3 +711,5 @@ wholesalefine.com
 wholesaleonlinemart.com
 worldcoinpay.com
 yourmonsterbeats.com
+
+full:beatselectronicsllc.tt.omtrdc.net
```
Comment (MuenYu) [inline data/beats:715]: unknow domain.

Comment (EpLiar) [reply data/beats:715]: > unknow domain.

https://www.beatsbydre.com/

**Review by MuenYu — APPROVED** [review]

**Review by MuenYu — APPROVED** [review]

File: data/apple [hunk 2]
```diff
@@ -409,30 +443,45 @@ appleid-applemx.us
...
+airtunes.com
 airtunes.info
+airtunes.net
 alchemysynth.com
 ap0le.com
+aple.com
+apole.com
+app0le.com
+appl-e.com
+appl.com
```
Comment (KevinZonda) [inline data/apple:482]: this should be insterted before apple-e.com

File: data/apple [hunk 3]
```diff
@@ -585,32 +654,47 @@ livephotos.cn @cn
 livephotos.com.cn @cn
 livephotos.tv
 lojaiphone.com.br
+mac-mini.com
+mac.com
```
Comment (KevinZonda) [inline data/apple:658]: This should insterd before mac-mini.com

Comment (EpLiar) [reply data/apple:658]: > This should insterd before mac-mini.com

按照字典序，`-` 比 `.` 小

**Pull request merged**